src/main/java/uk/gov/register/db/IndexQueryDAO.java

```diff
@@ -237,5 +240,14 @@
             "  index_entry.\"key\"  " +
             "order by  " +
             "  ien  ";
+    
+    String recordCountQuery = "select " +
```

Comment:
Even though the query is a lot shorter than the others, I still think its easier to read with line breaks.